File: c/iniciante/1017.c
/* title: Gasto de Combustível
 * number of the problem: 1017
 * date: 18/09/2020
 * author: ltsdw
 */

#include <stdio.h>
#define GMPK 12

int main(void) {
    double t, d;
    scanf("%lf", &t);
    scanf("%lf", &d);

    printf("%.3f\n", ((t * d) / GMPK));

    return 0;
}

File: haskell/iniciante/1017.hs
-- title: Gasto de Combustível
-- number of the problem: 1017
-- date: 21/09/20
-- author: ltsdw

import Text.Printf

gpmk :: Double
gpmk = 12.0

main :: IO ()
main = do
    t' <- getLine
    d' <- getLine

    let t = read t' :: Double
    let d = read d' :: Double

    printf "%.3f\n" ((t * d) / gpmk)

